axelor-account/src/main/java/com/axelor/apps/account/service/moveline/MoveLineService.java

```diff
@@ -49,4 +50,10 @@ void applyCutOffDates(
       MoveLine moveLine, Move move, LocalDate cutOffStartDate, LocalDate cutOffEndDate);
 
   BigDecimal getCutOffProrataAmount(MoveLine moveLine, LocalDate moveDate);
+
+  MoveLine clearAnalyticAccounting(MoveLine moveLine);
+
+  MoveLine printAnalyticAccount(MoveLine moveLine, Company company) throws AxelorException;
+
+  MoveLine checkAnalyticMoveLineForAxis(MoveLine moveLine);
```

Comment:
Same as before, please add in a specific analytic related service (along with other methods if there are)